python/client/cortex/client.py

```diff
@@ -144,11 +339,11 @@ def create_api(
             if not is_python_set:
                 conda_packages = [f"python={actual_version}", "pip=19.*"] + conda_packages
 
-        if len(requirements) > 0:
+        if requirements is not None:
```

Comment:
Should we take the branch if it's not none and has length > 0? e.g. `if requirements`? (same question for conda packages)